providers/sample/src/java/org/sakaiproject/provider/user/SampleCandidateDetailProvider.java

```diff
@@ -263,7 +325,7 @@ public boolean useInstitutionalAnonymousId(Site site) {
 	@Override
 	public boolean isInstitutionalNumericIdEnabled(Site site) {
 		try {
-			return (serverConfigurationService.getBoolean(SYSTEM_PROP_USE_INSTITUTIONAL_NUMERIC_ID, false)
+			return (serverConfigurationService.getBoolean(SYSTEM_PROP_USE_INSTITUTIONAL_NUMERIC_ID, true)
```

Comment:
Should these be enabled by default?

Reply:
Yes, this is enabled by default, as documented in the jira